tests/console/cpio.pm

```diff
@@ -0,0 +1,67 @@
+#SUSE"s openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Package: cpio
+# Summary: Create and extract archives with cpio, including all the supported archive formats
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Define the archive formats that will be used
+    my @formats = ("bin", "odc", "newc", "crc", "tar", "ustar", "hpbin", "hpodc");
+
+    # Download a folder with some files that will be used for the test purpose
+    my $testdatadir = "/usr/share";
+    assert_script_run "cd $testdatadir";
+    assert_script_run "wget --quiet " . data_url('console/cpio/topack.tar.gz');
+    assert_script_run "tar -xzvf $testdatadir/topack.tar.gz";
+
+    assert_script_run "mkdir /tmp/archive";
+
+    # Copy the files into an archive using the different archive formats
+    foreach my $format (@formats) {
+        assert_script_run "cd $testdatadir/topack && ls | cpio -o -H $format -O /tmp/archive/test_$format 2> /dev/null";
```

Comment:
Why `2> /dev/null` ?

Reply:
let's say leftovers from the testsuite, I will remove it

Reply:
I was just curious why just stderr, It should be better with all output. Thanks